src/components/users/userProfile/UserProfile.js

```diff
@@ -3,33 +3,50 @@ import { useState, useEffect } from 'react';
 import client from '../../../utils/client';
 import { useParams } from 'react-router-dom';
 import Header from '../../Header/Header';
-
+import './style.css'
 
 const UserProfile = () => {
-
   const [profile, setProfile] = useState('');
+  const [profileImg, setProfileImg] = useState('');
   const { id } = useParams();
-  let role = profile.role
+  let role = profile.role;
 
   useEffect(() => {
-      client
-        .get(`/user/${id}`)
-        .then((res) => {
-          setProfile(res.data.data.user);
-        })
-        .catch((err) => console.log(err.response));
+    handleProfile();
   }, [id]);
 
+  const handleProfile = () => {
+     client
+      .get(`/user/${id}`)
+      .then((res) => {
+        setProfile(res.data.data.user);
+        const firstName = res.data.data.user.firstName
+        const lastName = res.data.data.user.lastName
+        fetch(
+          `https://ui-avatars.com/api/?name=${firstName}+${lastName}`
+        )
+        .then(res=> { setProfileImg(res.url)} )
+      })
+      .catch((err) => console.log(err.response));
+  };
+
   return (
     <div>
-      <Header role={role}/>
+      <Header role={role} />
       <h1>User Profile</h1>
+      <div className="profile">
+      <div className="profile-img">
```

Comment:
Indent your markup 😭 😜

Reply:
Apologies! Just formatted it again and re submitted with the round img.